CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # How to contribute
 
-In this repo we follow the conventions defined in Vincent Driessen's **[git branching model](http://nvie.com/posts/a-successful-git-branching-model/)**. See **[git-flow](https://github.com/nvie/gitflow)** if you want some git extensions to help you with the mentioned model.
+In this repo, Feature Branch Workflow is the way we work with GIT. Basically this is based on creating branches that start from the master called feature branch. These branches have a single purpose, that is, they are created to add a specific functionality, to correct a bug, to delete deprecated functionality etc ... They will always be used for something specific. Once you complete what is being done in the feature branch, this functionality must be joined to master again and thus repeat the cycle during development.
 
 We **try** to use [Semantic Versioning](http://semver.org) and **Scrum** in our software development processes _as much as possible_. The [issues](https://github.com/iporaitech/pwr2-docker/issues) are grouped into [milestones](https://github.com/iporaitech/pwr2-docker/milestones) and most of the times will be labeled as: **Feature**, **Bug**, **Task**.  
```

Comment:
Below this paragraph add the following:

If you're in our organization and have rights to push to this repository, we recommend creating a branch **per issue** using the following naming conventions:

1. Start the branch name with the issue number, i.e.: 23-fix-logout-error.
2. If the issue is labeled with **Feature** add `-feature-` after the issue number, i.e.: 23-feature-login-with-facebook.
3. If you need to do some work specific for a **release** or a **hotfix** create a corresponding branch, i.e.: **release-v0.5.0** or **hotfix-v0.5.1**

If you don't have rights to push directly to this repo but still want to contribute, just send us a pull request to master, explaining in it what issues it solves. 

Always try to write meaningful commit messages referencing/closing issues.

We use **master** for our main/default branch.  When a pull request is accepted, we rebase it to master so we can keep a linear history. When we have code ready to be release to "production", we create a annotated  `git tag ...` which can be seen in the [releases](https://github.com/iporaitech/pwr2-docker/releases) section of this repo. This section also serves as our CHANGELOG.